backend/internal/storage/storage.go

```diff
@@ -10,7 +10,12 @@ type UserRepository interface {
 	GetUserByID(id string, ctx context.Context) (*models.User, error)
 }
 
+type SpotifyRepository interface {
+	GetPlatnmPlaylist(ctx context.Context) ([]*models.User, error)
```

Comment:
take advantage of zero values so your application doesn't crash catastrophically with an unnecessary nil pointer
```suggestion
	GetPlatnmPlaylist(ctx context.Context) ([]models.User, error)
```